src/shared/components/query/QueryStore.ts

```diff
@@ -1766,6 +1772,33 @@ export class QueryStore {
         );
     }
 
+    @computed get filteredMrnaProfiles() {
+        return this.getFilteredProfiles('MRNA_EXPRESSION');
+    }
+
+    @computed get shouldSelectDefaultMrnaProfile() {
+        // select default mRNA profile if no profiles are currently selected
+
+        if (Object.keys(this.selectedProfileIdSet).length > 0) {
+            return false;
+        }
+
+        const hasMutationProfile =
+            this.getFilteredProfiles('MUTATION_EXTENDED').length > 0;
```

Comment:
there are other profile types besides MUTATION_EXTENDED and MRNA so I suspect this logic is too narrow.  maybe what we really mean to do is if there is only ONE profile always use it?  or never have zero selected profiles?

Reply:
sorry for the delay, I've edited the logic so it checks for when there is only one profile, then use it. Please let me know if this change doesn't look good to you